src/nav/root.yml

```diff
@@ -54,16 +54,18 @@ pages:
     path: nrql
   - title: Service Level Management
     path: service-level-management
-  - title: Security 
+  - title: Security
   - title: New Relic IAST
-    path: iast 
+    path: iast
   - title: Vulnerability Management
-    path: vuln-management 
+    path: vuln-management
   - title: Latest updates
   - title: Release notes
     path: release-notes
   - title: What's new?
     path: whats-new
+  - title: End-of-life updates
```

Comment:
```suggestion
  - title: End-of-life announcements
```

I was worried that this would be too many characters, but I think it'd be good to match the page title now that I see it in preview